### DISCUSSION TITLE
Metadata URI on printed edition

### DISCUSSION THREAD

**Discussion by katewayo** [metaplex-foundation/js#159] [Q&A]
Hi @lorisleiva! Hope all is well. I'm running into an issue and I'm hoping you can give me some guidance. 

I'd like to metadata URI on a printed edition to be different from the original edition as this would allow certain operations to be specific to the edition without having the on-chain context. For example, without something like this, the generated external URL on the printed edition has no information to distinguish it from the original edition and that makes edition-specific behaviors when launched from a wallet difficult to implement.

I tried to update the metadata URI after the print, but I get an mutability error when I do that. Can I (or will I) be able to specify a new metadata URI when printing or be allowed to mutate the metadata URI on the printed edition? Any guidance would be appreciated? Thanks!

**Comment by lorisleiva** [comment]
Hi there 👋 

By definition, a printed edition is an exact copy of the original edition that cannot be changed. That means when printing a new edition, the Token Metadata Program will explicitly set the "is_mutable" field to "false" to ensure it cannot vary from the original edition.

That being said, the on-chain printed edition account keeps track of the edition number. You can use that to differentiate your printed edition and implement your custom logic.

I hope this helps.

**Reply by katewayo** [reply]
I ended up going a different direction with editions (partly because I need to future-proof for multi-chain cases). 

That said, I think there's an issue with implementing custom logic as you describe. For example, since the printed edition has the exact same metadata, how do you distinguish being launched for any specific edition when the external URL for the NFT is launched from a wallet? 

Does that make sense?

**Reply by lorisleiva** [reply]
Yes, unfortunately, if you need each printed edition to have its own Metadata URI then this is not a good use case for printed editions.

**Reply by katewayo** [reply]
Got it. That's the conclusion I came to and therefore went in a different direction. Thanks as always for your help and insight!
